src/main/scala/cromwell/engine/backend/sge/SgeBackend.scala

```diff
@@ -140,7 +140,8 @@ class SgeBackend extends Backend with SharedFileSystem with LazyLogging {
     logger.info(s"$tag SGE job completed (rc=$jobRc)")
     (jobRc, backendCall.stderr.toFile.length) match {
       case (r, _) if r == 143 => AbortedExecution // Special case to check for SIGTERM exit code - implying abort
-      case (r, _) if r != 0 => FailedExecution(new Throwable(s"$tag SGE job failed because of non-zero return code: $r"), Option(r))
+      case (r, _) if r != 0 && backendCall.call.failOnRc =>
```

Comment:
To be pedantic (cuz I roll like that) the ticket says non-zero exit code. @scottfrazer asked if we could do it as a whitelist of codes and the author of the ticket said that was AOK.